Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM docker.io/snyk/snyk:linux@sha256:91a21f9dfd09a93126d6b066f86f995013148253d9
 # Note that the version of OPA used by pr-checks must be updated manually to reflect dependabot updates
 # To find the OPA version associated with conftest run the following:
 # podman run --rm -ti ${NEW_IMAGE}  --version
-FROM docker.io/openpolicyagent/conftest:latest@sha256:93c0fccb97538b24ecb8492c5c988f39d27ec7108d4ce99f217677ad50f4271e as conftest
+FROM docker.io/openpolicyagent/conftest:latest@sha256:7886089bde3c924b0205b10aa299057c7cd2e3cc6dba0d8121ba12964a5117dc as conftest
```

Comment:
I could have sworn that we were supposed to be using a version tag instead of `latest` here, @Josh-Everett did something get reverted in #127 ?